shared/reducers/config.js

```diff
@@ -16,8 +16,10 @@ export default function(state: Types.State = initialState, action: ConfigGen.Act
         appFocusedCount: state.appFocusedCount,
         configuredAccounts: state.configuredAccounts,
         daemonHandshakeState: state.daemonHandshakeState,
+        daemonHandshakeVersion: state.daemonHandshakeVersion,
```

Comment:
when does resetStore get called again? perhaps we shouldn't merge in the versions, to keep them monotonically increasing

Reply:
it gets called when you log out. we need to keep this as there could be things in flight we care about